editor/question-ctrl.js

```diff
@@ -20,6 +20,13 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
 	$scope.questionData = {'questionMaxScore': 1};
 	$scope.questionData.isShuffleOption = false;
 	$scope.questionData.isPartialScore = true;
+  $scope.templateIcons = [];
+  _.each($scope.templatesType, function(template, key){
```

Comment:
This should be always in the init method. Functional logic always should be within the function. Only variables can be defined outside.